controllers/chat/group/groupAddMembers.js

```diff
@@ -96,7 +96,7 @@ const groupAddMembers = async (req, res) => {
         await BetterSocialCore.chat.updateBetterChannelMembers(
           channelToAdd,
           responseChannel,
-          true,
+          false,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `true` to `false` in the third argument of `updateBetterChannelMembers` function call might affect the functionality depending on how this boolean flag is used inside the function. If it's used to determine whether to update the channel members or not, then setting it to `false` could potentially disable updating the channel members for group channels. Please ensure that this is the intended behavior.
```diff
-          true,
+          false,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->